src/PlaylistController.js

```diff
@@ -0,0 +1,66 @@
+// import assert from 'assert';
+import GenericController from './GenericController';
+import Playlist from './Structs/Playlist';
+
+export default class PlaylistController extends GenericController {
+  constructor(...args) {
+    super(...args);
+    if (window.APPCONTEXT && window.APPCONTEXT.Go && window.APPCONTEXT.Go.h && window.APPCONTEXT.Go.h.length) {
+      this._playlists = Object.assign({}, window.APPCONTEXT.Go.h[0].Hi);
+      this._watchPlaylistObject();
+    } else {
+      this._playlists = {};
+      console.error('Could not listen to playlist changes in the expected location'); // eslint-disable-line
```

Comment:
In that case, then maybe we shouldn't do a `console.warn` either. As a developer, I would prefer to avoid noise I can't fix =/